testapp/admin.py

```diff
@@ -17,6 +20,16 @@ class UserAdmin(SharingConfigsExportMixin, SharingConfigsImportMixin, admin.Mode
     sharing_configs_export_form = ExportToForm
     sharing_configs_import_form = ImportForm
 
-    def get_sharing_configs_export_data(self, obj: object) -> str:
-        """current state: return list"""
-        return export_apps()
+    def get_sharing_configs_export_data(self, obj: object) -> bytes:
+        """return  django user model object as a byte like object"""
+
+        user = get_object_or_404(User, id=obj.id)
+        user_dict = model_to_dict(user)
+        dump_json_user = json.dumps(user_dict, sort_keys=True, default=str)
+        return dump_json_user.encode("utf-8")
+
+    def get_sharing_configs_import_data(self, content: bytes) -> object:
+        """current state: return bytes"""
+        decoded_content_utf = content.decode("utf-8")
+        obj = base64.b64decode(decoded_content_utf).decode("utf-8")
```

Comment:
Why do you base64 encode the file which you receive?

Reply:
Made it more complicated then it probably is. Changed in import mixin and  testapp implementation of  `get_sharing_configs_import_data`